### ISSUE TITLE
[MU4 Issue] Preferences > Advanced - Change text descriptions to user friendly

### ISSUE THREAD

**Issue by lengthwave** [musescore/MuseScore#11440]
**Describe the bug**
In Preferences > Advanced the preference descriptions are developer reference vs. user friendly descriptions.

**To Reproduce**
1. Go to Preferences > Advanced
2. See preference descriptions

**Expected behavior**
Preferences descriptions should be clear and user friendly in order to be immediately understood and less intimidating.

Example: 

_engraving/colors/voice1_

becomes....

_Voice 1 color_

**Screenshots**
<img width="1022" alt="Screen Shot 2022-04-27 at 8 54 00 PM" src="https://user-images.githubusercontent.com/30370609/165590008-219645be-6048-440f-bb75-b28a6a2ac1b9.png">

**Platform information**
OS: macOS 12.2
Arch.: x86_64
MuseScore version (64-bit): 4.0.0-2230826620
revision: github-musescore-musescore-b10e37c

**Comment by lengthwave** [comment]
@Tantacrul - FYI

**Comment by Tantacrul** [comment]
Yes, these should just be labelled the way they are in the main score.

**"Single-click to open a palette"**
**"Voice 1 colour"**

etc.

**Comment by donarturo11** [comment]
> Yes, these should just be labelled the way they are in the app itself.
> 
> **"Single-click to open a palette"** **"Voice 1 colour"**
> 
> etc.

Colour or color. As I know, that colour is in en_GB. Are we not using as base en_US, so color instead colour?

**Comment by cbjeukendrup** [comment]
@donarturo11 Yes, you're right, it should be `color` in the code / as the base language.

**Comment by the-next-ceo** [comment]
is it fixed?

**Comment by Tantacrul** [comment]
Not yet. It's a P3 so probably not one we'll get around to immediately
